### PR TITLE
add-cbr-module

### PR DESCRIPTION
**Pull request by Aayush-Abhyarthi**
### Description

add-cbr-module
#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [ ] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Review by Khuzaima05 — APPROVED** [review]

File: examples/complete/main.tf [hunk 1]
```diff
@@ -22,6 +22,36 @@ module "key_protect_all_inclusive" {
...
+  source           = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cbr//cbr-zone-module?ref=v1.2.0"
+  name             = "${var.prefix}-VPC-network-zone"
+  zone_description = "CBR Network zone representing VPC"
+  account_id       = data.ibm_iam_account_settings.iam_account_settings.account_id
+  addresses = [{
+    type  = "vpc", # to bind a specific vpc to the zone
+    value = ibm_is_vpc.example_vpc.crn,
+  }]
+}
+
```
Comment (ocofaigh) [inline examples/complete/main.tf:54]: You have created a CBR zone, but you never passed any BCR rule to the data engine module?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Please update the `examples/complete/README.md` file. Update the title to `Complete example with BYOK encryption and CBR rules` and also add a bullet point saying:
- A context-based restriction (CBR) rule to only allow Data Engine to be accessible from within the VPC.

**Comment by Aayush-Abhyarthi** [comment]
Data engine does not support CBR...?
[https://cloud.ibm.com/docs/account?topic=account-context-restrictions-whatis#cbr-adopters](url)

**Comment by ocofaigh** [comment]
@Aayush-Abhyarthi Lets try and find out internally when support will be added

**Comment by ocofaigh** [comment]
Data engine does not support CBR yet so closing this PR. Keep an eye on updates in https://github.com/terraform-ibm-modules/terraform-ibm-data-engine/issues/9

**Pull request closed without merging**